tests/common.go

```diff
@@ -4044,7 +4044,7 @@ func ParseFlags() {
 		}
 		if testsetID != 0 {
 			dash.TestSetID = testsetID
-			os.Setenv("TESTSET-ID", fmt.Sprint(testsetID))
+			os.Setenv("DASH_UID", fmt.Sprint(testsetID))
```

Comment:
why do we need to set environment variable? The dash object should use it if pass `aetos-uuid` as a aparameter

Reply:
This is added when we are using separate binaries , where it is creating different testset id for each test, this can be removed now as we are using only one binary for all the tests. 
@CharudathGopal you can raise a ticket in my name for this clean up.Will make the changes and test it

Reply:
I dont see a use of it at this point, but this is added to reuse the UID when trying multiple runs in the same context.

@lsrinivas-pure are we using this anywhere as of now ?